app/Jobs/PollForMediaWikiJobsJob.php

```diff
@@ -10,7 +10,7 @@
 
 class PollForMediaWikiJobsJob extends Job implements ShouldQueue, ShouldBeUnique
 {
-    public $timeout = 3600;
+    public $timeout = 1800;
```

Comment:
Currently this takes something around 3 to 7 minutes in production (depending on current load), so I think it's fine to lower this while we're at it.